pop-api/examples/filtered-call/lib.rs

```diff
@@ -0,0 +1,110 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
```

Comment:
This shouldn't be in `examples` as it is not showing anything useful to builder. As they are only being used for tests, suggest moving to a `tests/contracts` directory in the root, ala Rust integration tests.